rbkit-lib/cpu/cpumapping.h

```diff
@@ -0,0 +1,28 @@
+#include <QString>
+
+#ifndef MAPPING
+#define MAPPING
+
+inline namespace MAPS {
+    enum Map {
+        event_type,
```

Comment:
this mapping is already present here: https://github.com/code-mancers/rbkit-client/blob/master/rbkit-lib/rbeventparser.h#L11